nupic/algorithms/anomaly.py

```diff
@@ -157,6 +144,30 @@ def compute(self, activeColumns, predictedColumns,
     return score
 
 
+############################################
+# implementations of compute() function
+  def computeRaw(self, active, prevPredicted, inputValue=None, timestamp=None):
+    return computeRawAnomalyScore(active, prevPredicted)
+
+  def computeLikelihood(self, active, prevPredicted, inputValue, ts=None):
+    """Anomaly computed using the anomaly_likelihood score, 
+       which models probability of (input, anomalyScore) pair.
+    """
+    assert isinstance(self._likelihood, AnomalyLikelihood)
+    if inputValue is None:
+      raise ValueError("Selected anomaly mode 'Anomaly.MODE_LIKELIHOOD' "
+                       "requires 'inputValue' as parameter to compute() method. ")
+    rawScore = computeRawAnomalyScore(active, prevPredicted)
+    probability = self._likelihood.anomalyProbability(inputValue, rawScore, ts)
```

Comment:
No need to save the value in a local variable if the next statement simply returns it. More concise as: 

`return self._likelihood.anomalyProbability(inputValue, rawScore, ts)`